test/pageobjects/page.ts

```diff
@@ -0,0 +1,15 @@
+import { browser } from '@wdio/globals'
+
+/**
+* main page object containing all methods, selectors and functionality
+* that is shared across all page objects
+*/
+export default class Page {
+    /**
+    * Opens a sub page of the page
+    * @param path path of the sub page (e.g. /path/to/page.html)
+    */
+    public open (path: string) {
+        return browser.url(`http://localhost:3000/${path}`)
+    }
+}
```

Comment:
Idea: Substitute static url to value from general `wdio.conf.ts`. Requires change of tsconfig as described below:
```suggestion
import { browser } from '@wdio/globals'
import { config } from '../../wdio.conf.ts'
/**
* main page object containing all methods, selectors and functionality
* that is shared across all page objects
*/
export default class Page {
    /**
    * Opens a sub page of the page
    * @param path path of the sub page (e.g. /path/to/page.html)
    */
    public open (path: string) {
        return browser.url(`${config.baseUrl}/${path}`)
    }
}
```

Reply:
changed in 82c05c6f460f96c3e0d3ecf720c2b7ffaeb67bb1